data/locale/sv.ts

```diff
@@ -386,39 +386,39 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>Discrete progression</source>
-        <translation>Diskret talföljd</translation>
+        <translation>Diskret tongång</translation>
     </message>
     <message>
         <source>Linear progression</source>
-        <translation>Linjär talföljd</translation>
+        <translation>Linjär tongång</translation>
     </message>
     <message>
         <source>Cubic Hermite progression</source>
-        <translation>Cubic Hermite talföljd</translation>
+        <translation>Cubic Hermite-tongång</translation>
     </message>
     <message>
         <source>Tension value for spline</source>
-        <translation>Spänning i mönstrets spline</translation>
+        <translation>Spänningsvärde för spline</translation>
     </message>
     <message>
         <source>A higher tension value may make a smoother curve but overshoot some values. A low tension value will cause the slope of the curve to level off at each control point.</source>
-        <translation>Högre spänning ger en mjuk kurva som ibland missar individuella punkter. Med lägre spänning planar kurvan ut nära punkterna.</translation>
+        <translation>Ett högre spänningsvärde kan ge en mjukare kurva, men som skjuter över några värden. Ett lågt spänningsvärde kommer att få kurvans lutning att plana ut vid varje kontrollpunkt.</translation>
     </message>
     <message>
         <source>Click here to choose discrete progressions for this automation pattern.  The value of the connected object will remain constant between control points and be set immediately to the new value when each control point is reached.</source>
-        <translation>Klicka här för att aktivera diskret talföljd. Värdet är konstant mella kontroll punkter och ändras direkt när en ny kontrollpunkt nås.</translation>
+        <translation>Klicka här för att välja diskret tongång för det här automatiska mönstret. Värdet för det anslutna objektet kommer att hållas konstant mellan kontrollpunkter och blir direkt inställd till det nya värdet när varje kontrollpunkt nås.</translation>
     </message>
     <message>
         <source>Click here to choose linear progressions for this automation pattern.  The value of the connected object will change at a steady rate over time between control points to reach the correct value at each control point without a sudden change.</source>
-        <translation>Klicka här för att aktivera linjär talföljd. Värdet ändras vid en stadig takt mellan kontrollpunkter för att gradvis nå nästa värde.</translation>
+        <translation>Klicka här för att välja linjär tongång för det här automatiska mönstret. Värdet av det anslutna objektet kommer att förändras i en stadig takt över tiden mellan kontrollpunkter för att nå det korrekta värdet vid nästa kontrollpunkt utan en plötslig ändring.</translation>
```

Comment:
While I agree that "talföljd" (number sequence) is a bad translation (it's accurate to what's going on code-wise, I think, but the user shouldn't know/care about that), I don't like tongång either, as I've only seen it used to refer to melodies. Some googling and various translation sites show that it can be used to mean progress/progression though, so you're correct. A much bigger issue I have with this line is the translation of "automation pattern" to "automatiska mönstret". This suggests that the pattern itself is automatic, when it is really used to "automate" other things. I'm not sure of a better translation, unfortunately.